src/agent/src/config.rs

```diff
@@ -78,6 +82,8 @@ pub struct AgentConfigBuilder {
     pub server_addr: Option<String>,
     pub unified_cgroup_hierarchy: Option<bool>,
     pub tracing: Option<bool>,
+    pub aa_kbc_params: Option<String>,
```

Comment:
Another issue is that current the `aa_kbc_params` is no longer used in image-rs anymore for Kata-CC. In short, we do not need this parameter in kata-agent anymore for non-peer-pod scenarios.

In old times, we need this parameter from kata-agent to call `image-rs` as a parameter to decrypt image. The `aa_kbc_params` will be deliver via `image-rs` -> `ocicrypt-rs` --ttrpc-> `Attestation Agent` and launch coresponding `kbc` with `kbs url`.

Now, after [the refactoring](https://github.com/confidential-containers/guest-components/pull/427), the image decryption is [moved to CDH](https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/hub/src/hub.rs#L56-L65). And CDH itself will [read kernel cmdline or peer pod config](https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/kms/src/plugins/kbs/mod.rs#L36) to find `aa_kbc_params` for now as a legacy way before we have initdata mechanism to specify that in a configuration file.

About setting `aa_kbc_params` in configuration file, please see [the issue](https://github.com/confidential-containers/guest-components/issues/431).

A left question is whether CDH is also be built in peer-pod's guest image? If so, the image decryption can work without setting this field of kata-agent. Only need to specify in CDH, which is already implemented.

cc @mkulke @bpradipt

Reply:
yes, CDH is part of a peerpod image by default. At the moment a `aa_kbc_params` string is retrieved from a kata-agent config file. Some specific [peerpod logic](https://github.com/confidential-containers/cloud-api-adaptor/blob/72ea7fa2a5e802ac9862ecde10b64c1a44c2c250/cmd/process-user-data/update.go#L66-L73) is templating the value in the agent config file. 

To my knowledge there is now good reason why it should be like that, beyond legacy (@stevenhorsman can you confirm). It could be an AA configuration file or an env.

Reply:
Yeah - from what I can see `aa_kbc_params` doesn't need to be passed to the kata-agent now, so we can drop it from this PR. We just need to ensure that peer pods can configure it in a place that CDH can read it, either via initData, process-user-data, (or backed in as a last resort).

Reply:
removed, thanks.